README.md

```diff
@@ -173,6 +175,35 @@ Use Google doc to enable in-line comments and suggestions.
 
 *Allocate enough time for your reading, discuss as a group, write the summary carefully, and finally, include key observations from the class discussion.*
 
+### Post-Presentation Panel Discussion 
+To foster a deeper understanding of the papers and encourage critical thinking, each lecture will be followed by a panel discussion. 
+This discussion will involve three distinct roles played by different student groups, simulating an interactive and dynamic scholarly exchange.
+Roles and Responsibilities:
+
+1. The Authors:
+- Group Assignment: The group that presents the paper will play the role of the paper's authors.
+- Responsibility: As authors, you are expected to defend your paper against critiques, answer questions, and discuss how you might improve or extend your research in the future, akin to writing a rebuttal during the peer-review process.
+
+
+2. The Reviewers:
+- Group Assignment: Each group will **sign up for one slot to play the role of reviewers**.
+- Responsibility: Reviewers critically assess the paper, posing challenging questions and highlighting potential weaknesses or areas for further investigation. 
+Your goal is to engage in a constructive critique of the paper, simulating a peer review scenario.
+
+
+3. The Industry Sponsors:
```

Comment:
This seem far-fetched and I don't see what they'd add. We can play this role if needed or just drop it.